src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -16,30 +16,263 @@ See License.txt for details.
 #include <vtkSmartPointer.h>
 #include <vtksys/CommandLineArguments.hxx>
 
+
+#include "PlusTrackedFrame.h"
+#include "vtkPlusTrackedFrameList.h"
+#include "vtkImageCast.h"
+
 //----------------------------------------------------------------------------
 
+
+
 int main(int argc, char **argv)
 {
-  int verboseLevel = vtkPlusLogger::LOG_LEVEL_UNDEFINED;
+	bool printHelp = false;
+	vtksys::CommandLineArguments args;
+
+
+	//Set up variables for command line arguments
+	std::string inputFileName;
+	std::string configFileName;
+	std::string outputFileName;
+	std::string linesImageFileName;
+	std::string intermediateImageFileName;
+	std::string processedLinesImageFileName;
+
+	double gaussianStdDev = NULL;

```

Comment:
These many cmd line parameters are too much. Too much effort for the user, too long command line, too much work to add more parameters. You may keep 1-2 frequently changing key parameters in command-line arguments, but all the rest should be read from a config file - the same way as it is done for volume reconstruction parameters.